yoga/Yoga.h

```diff
@@ -38,6 +38,16 @@ typedef struct YGSize {
   float height;
 } YGSize;
 
+typedef struct YGValue{
```

Comment:
I would like to do this, but setting everything up on my maschine would be a huge job. As I have no linux runtime currently available, and also no clang localy. It would be greate if you could run this, after your import, if you don't mind. Same for enum.py, currently I have no python installed. If not I'll try my best to set this up locally, could take a while (the clang environment).